### DISCUSSION TITLE
I am getting ModuleNotFoundError: No module named 'airflow.www_rbac' in below python script.

### DISCUSSION THREAD

**Discussion by Rupam31592** [apache/airflow#30366] [Q&A]
This code was written in according to airflow 1.10.10 and now we are using airflow 2.5.1. Please help!

This code is supposed to sync users from azure AD. Please suggest me also in airflow 2.5.1 how we can sync users with azure AD.

[usersync.txt](https://github.com/apache/airflow/files/11109823/usersync.txt)

**Comment by Rupam31592** [comment]
Hello @potiuk do you have any idea.

**Comment by potiuk** [comment]
Yes. Update the code to be compliant with Airflow 2. I am not going to even get close to Airflow 1.10-relevant code 3 years after it reached EOL - you are on your own - but feel free to take a look at our repository and compare 1.10 to 2.

**Reply by Rupam31592** [reply]
Hey @potiuk  I just need the new module name in replace of www_rbac. Which I am not able to find out properly. Thats the reason to reach out here. If you also notice the code consists few very simple python functions. But the only dependency with airflow 1 is this module 
from airflow.www_rbac.app import cached_appbuilder. and we are getting exception here in line 70. appbuilder = cached_appbuilder()

I want suggestion from you on in airflow 2 how I can use appbuilder? Also if you can tell me if there are other ways to sync ad users with airflow.

Thanks,
Rupam

**Reply by potiuk** [reply]
No idea, really but maybe you can search for similar issues/others doing it
